phpstan-baseline.neon

```diff
@@ -200,6 +200,11 @@ parameters:
 			count: 1
 			path: src/Exception/BadResponseException.php
 
+		-
+			message: "#^Method GuzzleHttp\\\\Exception\\\\BadResponseException\\:\\:getResponse\\(\\) should return Psr\\\\Http\\\\Message\\\\ResponseInterface but returns Psr\\\\Http\\\\Message\\\\ResponseInterface\\|null\\.$#"
+			count: 1
+			path: src/Exception/BadResponseException.php
+
```

Comment:
I added this to baseline

IMHO it is a false positive.

Reply:
Might even be a bug

Reply:
No it is not... phpstan only looks at the docblock..  and does not check the whole flow of the code in order to understand that the response can never be null...

Reply:
What if you add a docblock to the function then?